public/pages/Alerts/containers/Alerts/Alerts.tsx

```diff
@@ -252,12 +290,20 @@ export class Alerts extends Component<AlertsProps, AlertsState> {
     this.setState({ flyoutData: alertItem ? { alertItem } : undefined });
   }
 
+  // Function to show the tooltip
+  showTooltip = (errorMessage) => {
```

Comment:
Where are we using this and the below method?

Reply:
addressed